src/misc/spinner.tsx

```diff
@@ -0,0 +1,55 @@
+import { RendererLike } from 'render-jsx';
```

Comment:
We already had a [loading component](https://github.com/Coding-blog/editor-frontend/blob/main/src/misc/loading.tsx), may I ask why we need this as well, since it seems to be doing the same thing?

Reply:
Two reasons:
- I thought it looks cool
- I didn't notice the other one at first

Reply:
Is this comment at the right spot?